src/Coordination/KeeperStorage.cpp

```diff
@@ -1129,6 +1129,12 @@ struct KeeperStorageRequestProcessor
         return {};
     }
 
+    virtual KeeperStorageBase::ResponsesForSessions
+    processWatches(const Storage & /*storage*/, int64_t /*zxid*/, KeeperStorageBase::Watches & watches, KeeperStorageBase::Watches & list_watches) const
+    {
+        return processWatches(watches, list_watches);
```

Comment:
Yeah, I don't really like this change either, it's necessary because processors don't save the state. For example, we can leave only the variant of the method with 4 parameters.

Reply:
left only the new variant of the method, please check is it okay?